infoschema/builder.go

```diff
@@ -130,7 +228,7 @@ func (b *Builder) ApplyDiff(m *meta.Meta, diff *model.SchemaDiff) ([]int64, erro
 		}
 
 		tmpIDs := tblIDs
-		if (diff.Type == model.ActionRenameTable || diff.Type == model.ActionRenameTables) && diff.OldSchemaID != diff.SchemaID {
+		if diff.Type == model.ActionRenameTable && diff.OldSchemaID != diff.SchemaID {
```

Comment:
seems we can keep the ActionRenameTables here, so we don't need to change the type name above right ?

Reply:
Nope, since we change the type name, so we can skip `ActionRenameTables` here. If that is too confusing, I could change it back.

Reply:
One thing I want to say is that is will be soon become complicated if there are more and more batching actions...

Reply:
For examples, every create table needs another `|| == model.ActionCreateTables`, and possibly other DDLs.